teamcity/hatch.toml

```diff
@@ -14,6 +14,7 @@ matrix.impl.env-vars = [
 ]
 matrix.python.scripts = [
   { key = "test", value = "_dd-test --ignore=tests/docker" },
+  { key = "test-cov", value = "_dd-test-cov --ignore=tests/docker" },
```

Comment:
Could we set this flag in a `conftest.py` or even `pyproject.toml`? That way it would apply always.

Reply:
I don't know 😅 Will wait for ofek on this one if that's ok for you

Reply:
yea, looking forward to Ofek's input! iiuc, putting it into config could mean we get rid of both that whole section!